apps/src/studio/skins.js

```diff
@@ -539,6 +540,7 @@ function loadHoc2015x(skin, assetUrl) {
   };
 
   skin.specialItemProperties = {
+    'hazard': { frames: 13, width: 100, height: 100, renderOffset: { x: 0, y: -25}, activity: 'watchActor', speed: constants.SpriteSpeed.VERY_SLOW }
```

Comment:
Conceptually I don't think that's right.  speed seems to be "possible maximum speed" and actual motion should be determined by direction.  The reason for this is that if we every gave the student control over a character like this and they changed the action to "walk" but the speed was zero, the character would essentially walk in place, which seems like a bug.